core/src/main/java/oracle/weblogic/deploy/util/WLSDeployArchive.java

```diff
@@ -952,22 +963,22 @@ public String addMimeMappingFile(File mimeMappingFile) throws WLSDeployArchiveIO
     }
 
     /**
-     * Add a Coherence configuration file to the archive.
+     * Add a Foreign Server binding file to the archive
      *
-     * @param clusterName the Coherence cluster name used to segregate the directories
+     * @param foreignServer the Foreign Server name used to segregate the directories
      * @param configFile  the file to add
-     * @return thje new location of the file to use in the model
+     * @return the new location of the file to use in the model
      * @throws WLSDeployArchiveIOException if an error occurs while archiving the file
-     * @throws IllegalArgumentException    if the file does not exist or the clusterName is empty or null
+     * @throws IllegalArgumentException    if the file does not exist or the foreignServer is empty or null
      */
-    public String addCoherenceConfigFile(String clusterName, File configFile) throws WLSDeployArchiveIOException {
-        final String METHOD = "addCoherenceConfigFile";
```

Comment:
Did you intend to remove addCoherenceConfigFile()?
It's still referenced in coherence_resources_discoverer.py